docs/cli.md

```diff
@@ -121,11 +128,28 @@ optional arguments:
   --run-config-search-max-preferred-batch-size RUN_CONFIG_SEARCH_MAX_PREFERRED_BATCH_SIZE
                         Max preferred batch size value that run config search should not go beyond that.
 ```
-Depending on the command line or YAML config options provided, the `profile` subcommand will either perform a manual or automatic search over perf analyzer and model config file parameters. For each combination of model config parameters (e.g. *instance count* and *dynamic batch size*), it will run tritonserver and perf analyzer instances with all the specified run parameters (client request concurrency and static batch size). It will also save the protobuf (.pbtxt) model config files corresponding to each combination in the **output model repository**. Model Analyzer collects various metrics at fixed time intervals during these perf analyzer runs. Each perf analyzer run generates a single measurement, which corresponds to a row in the output tables. After completing the runs for all configurations for each model, the Model Analyzer will save the measurements it has collected into the **checkpoint directory** as a *pickle* file. See the [Checkpointing](./checkpoints.md) section for more details on checkpoints
+Depending on the command line or YAML config options provided, the `profile`
+subcommand will either perform a manual or automatic search over perf analyzer
+and model config file parameters. For each combination of model config
+parameters (e.g. *instance count* and *dynamic batch size*), it will run
+tritonserver and perf analyzer instances with all the specified run parameters
+(client request concurrency and static batch size). It will also save the
+protobuf (.pbtxt) model config files corresponding to each combination in the
+**output model repository**. Model Analyzer collects various metrics at fixed
```

Comment:
Can you make `output_model_repository` a link to the corresponding config option?